pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -291,3 +297,92 @@ type SecretOrConfigMap struct {
 	// ConfigMap containing data to use for the targets.
 	ConfigMap *corev1.ConfigMapKeySelector `json:"configMap,omitempty"`
 }
+
+// RemoteWriteSpec defines the remote_write configuration for prometheus.
+// +k8s:openapi-gen=true
+type RemoteWriteSpec struct {
+	// The URL of the endpoint to send samples to.
+	URL string `json:"url"`
+	// +optional
+	RemoteTimeout model.Duration `json:"remote_timeout,omitempty"`
```

Comment:
How about unifying the tag format to `remoteTimeout`?

Reply:
ditto for `queue_config`.

Reply:
ok